iothub/device/src/InternalClient.cs

```diff
@@ -320,7 +320,6 @@ public async Task OpenAsync(CancellationToken cancellationToken)
             }
             catch (IotHubCommunicationException ex) when (ex.InnerException is OperationCanceledException)
```

Comment:
slightly tangential note (pending answer to the other question), but this block doesn't really do anything anymore, so might as well remove the `catch` block entirely.

Reply:
Which other question? Did you post on this PR? I agree, this catch block can be removed. It never did anything earlier as well.

Reply:
Yeah, I meant this one: https://github.com/Azure/azure-iot-sdk-csharp/pull/1955#issuecomment-839925262

Reply:
I see it now. It was not showing up till now.

Reply:
Fyi: That block never did anything so it is safe to remove at the moment.